SAP-SalesOrder-GetStatus-BAPI/workflow.json

```diff
@@ -0,0 +1,191 @@
+{
+    "definition": {
+        "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
+        "actions": {
+            "Condition": {
+                "actions": {
+                    "Post_adaptive_card_and_wait_for_a_response": {
+                        "inputs": {
+                            "body": {
+                                "body": {
+                                    "messageBody": "{\n  \"$schema\": \"http://adaptivecards.io/schemas/adaptive-card.json\",\n  \"type\": \"AdaptiveCard\",\n  \"version\": \"1.2\",\n  \"body\": [\n    {\n          \"type\": \"TextBlock\",\n          \"text\": \"Enter the sales order number incl. leading zeros:\",\n          \"weight\": \"Bolder\",\n          \"size\": \"Medium\"\n        },\n    {\n      \"type\": \"Input.Text\",\n      \"id\": \"salesOrderNumber\",\n      \"placeholder\": \"Enter Sales Order Number incl. leading zeros\",\n      \"errorMessage\": \"Sales Order Number is required\"\n    }\n  ],\n  \"actions\": [\n    {\n      \"type\": \"Action.Submit\",\n      \"title\": \"Submit\",\n      \"data\": {\n        \"action\": \"submitSalesOrderNumber\"\n      }\n    }\n  ]\n}",
+                                    "recipient": {
+                                        "to": "@triggerBody()?['from']?['user']?['id']"
+                                    },
+                                    "updateMessage": "Thanks for your response!"
+                                },
+                                "notificationUrl": "@{listCallbackUrl()}"
+                            },
+                            "host": {
+                                "connection": {
+                                    "referenceName": "teams"
```

Comment:
The error message generated by the test:
`Workflow "SAP-UserUnblock-SelfService-BAPI" Failed Validation: connection used in "referenceName": "teams" not found in manifest.json. Hint: Make sure the connection name is in the format <connectionName>_#workflowname#`

Since this is a connection, this has to be listed in the connections undermanifest.json. The error message is suggesting that you do not have a connection name (key) which is "teams".